config.rb.sample

```diff
@@ -38,6 +38,11 @@ $new_discovery_url="https://discovery.etcd.io/new?size=#{$num_instances}"
 # "core-01" through to "core-${num_instances}".
 #$instance_name_prefix="core"
 
+# Change the version of CoreOS to be installed
+# The default value is "current", which points to the current version
+# of the selected channel
+#$image_version = ""
```

Comment:
Following the convention already established in this file, this line should read `#$image_version="current"`